metaworld/policies/sawyer_dial_turn_v1_policy.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy, assert_fully_parsed, move
+
+
+class SawyerDialTurnV1Policy(Policy):
+
+    @staticmethod
+    @assert_fully_parsed
+    def _parse_obs(obs):
+        return {
+            'hand_pos': obs[:3],
+            'dial_pos': obs[3:],
```

Comment:
is this compatible with the new observation space?

Reply:
It is now updated! Take a look now.